src/components/LeftDrawerEvent/LeftDrawerEventWrapper.test.tsx

```diff
@@ -65,10 +66,7 @@ jest.mock('@mui/x-charts/PieChart', () => ({
 beforeEach(() => {
   setItem('FirstName', 'John');
   setItem('LastName', 'Doe');
-  setItem(
-    'UserImage',
-    'https://api.dicebear.com/5.x/initials/svg?seed=John%20Doe',
-  );
+  setItem('UserImage', <Avatar name="John Doe" alt="John Doe" />);
```

Comment:
@akhilender-bongirwar for test purposes, we were storing the image url in localstorage as if it was user's own profile pic

Now we could have used any image link over here, but we decided to go with using a url from Dicebear. So replacing this with Avatar component is redundant, you can remove this.

We needed Avatar component inside the `.tsx` files, not in the test files, unless you want to test some custom functionality.

Reply:
Ok. I have reverted all the Avatar components in test files where they are not required.